packages/web/src/pages/search-page/store/sagas.js

```diff
@@ -15,8 +15,8 @@ import { waitForBackendSetup } from 'store/backend/sagas'
 
 export function* getTagSearchResults(tag, kind, limit, offset) {
   const results = yield call(AudiusBackend.searchTags, {
-    searchText: tag.toLowerCase(),
-    minTagThreshold: 1,
+    query: tag.toLowerCase(),
```

Comment:
wild, so broken or you changed the api?

Reply:
Changed the api to match the types defined in libs